rules/linux/persistence_systemd_scheduled_timer_created.toml

```diff
@@ -0,0 +1,92 @@
+[metadata]
+creation_date = "2023/02/24"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/03/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects the creation of a systemd timer within any of the default systemd timer directories. Systemd timers
+can be used by an attacker to gain persistence, by scheduling the execution of a command or script. Similarly
+to cron/at, systemd timers can be set up to execute on boot time, or on a specific point in time, which allows
+attackers to regain access in case the connection to the infected asset was lost. 
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "New Systemd Timer Created"
+note = """## Triage and analysis
+
+### Investigating New Systemd Timer Created
```

Comment:
We can remove this one for now and do it on a dedicated PR, or we can merge it without the Investigation Guide tag.

Other than that, you can compare to existing guides and adjust the structure accordingly

Reply:
Adjusted the structure in new commit.